arielTROY.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "@arielTROY"
+project: "forblind"
+---
+
+# Project Name
+
+## Summary
+
+I'm going to build esp32 based smart sunglasses that help blind people get information with ai technologies about what's in front of them. 
+If it is text then we will use the ocr model and if it is an object we will use the image classification model.
+The purpose of the project is to allow everyone to create the glasses for the other cheaply and not like today a similar product costs about 1200 dollars. 
+I like the project because I know that there are people who really need a product in this style and do not necessarily have 1200 usd for an orcam
+
+## Plan
+
+First of all, order the desired ingredients.
+After that I will start expanding knowledge about tensorflow and different artificial intelligence models and their use on esp32  . 
+I will put all the components together and print a case on a resin 3D printer.
+I will create a short program that uses python and recognizes voice commands with the help of voice recognition ai model and then takes a picture and processes it and plays it on the device's speakers. 
+Finally, maybe added a user application in swift ui
+
+## Note
+Probably when the button on the device will be pressed then it will capture a photo and send the photo to mobile app that will procces the image with tensorflow and will read with tts what infront of you.
+but maybe i will test tensorflow for micro devices without phone and compare the latency and results between the two methods to see which better
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| bluetooth headphones|at home   | $0 |\
+| wires | at home  | $0 |
+| esp32cam board |https://www.amazon.com/Aideepen-ESP32-CAM-Bluetooth-ESP32-CAM-MB-Arduino/| $12.99 | free shipping  
```

Comment:
hiii :) can you please update this link? it leads to a page not found for me... is it this link? https://www.amazon.com/Aideepen-ESP32-CAM-Bluetooth-ESP32-CAM-MB-Arduino/dp/B08P2578LV

Reply:
updated